docs/kane-ai-using-parameters.md

```diff
@@ -41,4 +41,88 @@ import {YOUR_LAMBDATEST_USERNAME, YOUR_LAMBDATEST_ACCESS_KEY} from "@site/src/co
+
+<img loading="lazy" src={require('../assets/images/kane-ai/features/parameters/3.png').default} alt="Image" className="doc_img"/>
+
+## Step 3: Finalize the Parameter
+Reference any parameter using **$curly braces syntax** (e.g.,`${phone_number}`). Enter the name in its designated input field.
+
+<img loading="lazy" src={require('../assets/images/kane-ai/features/parameters/4.png').default} alt="Image" className="doc_img"/>
+
+## Step 4: Input Additional Data
+Input values for the age and phone number fields.
+Execute the test as usual.
+
+<img loading="lazy" src={require('../assets/images/kane-ai/features/parameters/5.png').default} alt="Image" className="doc_img"/>
+
+## Step 5: Submit and Review Parameters
+- Access the parameter list for navigation.
+- Click Submit to confirm the data inputs.
+- Finalize by clicking Finish Test.
+
+<img loading="lazy" src={require('../assets/images/kane-ai/features/parameters/6.png').default} alt="Image" className="doc_img"/>
+
+## Step 6: Test Summary Page
+Upon completing the test, review the Test Summary Page, where all utilized parameters are clearly displayed, and the corresponding generated code is readily available for seamless integration into future test scenarios.
+<img loading="lazy" src={require('../assets/images/kane-ai/features/parameters/7.png').default} alt="Image" className="doc_img"/>
+
+## Step 7: Navigate to Test Project
+Open the Test Project dashboard and go to the **Test Run** section.
+
+<img loading="lazy" src={require('../assets/images/kane-ai/features/parameters/8.png').default} alt="Image" className="doc_img"/>
+
+## Step 8: Create a New Test Run
+- Click **Create New Test Run** and name it (e.g., parameters test run).
+- Select the KaneAI-generated test cases.
+- Optionally, add a description or relevant tags.
+- Click **Create Test Run**.
+
+<img loading="lazy" src={require('../assets/images/kane-ai/features/parameters/9.png').default} alt="Image" className="doc_img"/>
+
+## Step 9: Add Test Cases and Data Sets
+- Select and add the desired test cases to the run.
+- Specify configurations (e.g., Windows 10 Chrome).
+- Attach the required data sets for the test run.
+
+<img loading="lazy" src={require('../assets/images/kane-ai/features/parameters/10.png').default} alt="Image" className="doc_img"/>
+
+## Step 10: Apply Data Sets
+- Ensure the data sets containing necessary parameters are pre-filtered and accessible.
+- Click **Apply** to confirm the Dataset selection.
+
+<img loading="lazy" src={require('../assets/images/kane-ai/features/parameters/11.png').default} alt="Image" className="doc_img"/>
+
+## Step 11: Save and Prepare for Execution
+- Save the test run with the selected parameters and Datasets. The system will create test instances ready for execution in a hybrid environment.
+
+<img loading="lazy" src={require('../assets/images/kane-ai/features/parameters/12.png').default} alt="Image" className="doc_img"/>
+
+## Step 12: Execute Tests with Parameters
```

Comment:
Group steps 7 to 12 and add then under "Using parameters for test runs" @amanchopra1905